rollup.config.js

```diff
@@ -0,0 +1,27 @@
+import resolve from 'rollup-plugin-node-resolve';
+import commonjs from 'rollup-plugin-commonjs';
+import uglify from 'rollup-plugin-uglify';
+import yargs from 'yargs';
+
+const args = yargs.argv;
+
+export default {
+  input: './lib/index.js',
+  output: {
+    file: `dist/index${args.min ? '.min' : ''}.js`,
+    format: 'umd',
+  },
+  name: 'ReactBeautifulDnd',
+  plugins: [
+    resolve({
+      browser: true,
+      main: true,
+    }),
+    commonjs({
```

Comment:
not sure what this one is for. I think just listing react as an external is enough

Reply:
`rollup-plugin-commonjs` complains that React does not export `'Component', 'Children', 'createElement'` as named exports. This was the solution I've found.

Reply:
Listing React as an external dependency leads to
`(!) Missing global variable name
Use options.globals to specify browser global variable names corresponding to external modules
react (guessing '_react')`
I'm not sure if that's fine, or should I include `globals: { react: 'React' }` to `rollup.config.js`?

Reply:
Yeah, I think you're going to have to do that. I've found you can automate the dependencies part by doing something like:

```js
const pkg = require('./package.json');
const skip = {
  ...pkg.dependencies,
  ...pkg.devDependencies,
  ...pkg.optionalDependencies,
  ...pkg.peerDependencies
};
```

Jason Miller (of Preact fame) also shows this [here](https://gist.github.com/developit/41f088b6294e2591f53b).